src/Auth/Models/User.php

```diff
@@ -148,6 +148,10 @@ public function afterLogin()
         $this->forceSave();
     }
 
+    public function onSuspend()
```

Comment:
@josephcrowell Please add a docblock here that describes the intention of the method.